SingularityUI/app/reducers/api/index.es6

```diff
@@ -50,11 +50,13 @@ import {
 
 import { FetchWebhooks } from '../../actions/api/webhooks';
 
+import { FetchAction as TaskHistory } from '../../actions/api/taskHistory';
+
 const user = buildApiActionReducer(FetchUser);
 const webhooks = buildApiActionReducer(FetchWebhooks, []);
 const slaves = buildApiActionReducer(FetchSlaves, []);
 const racks = buildApiActionReducer(FetchRacks, []);
-const request = buildKeyedApiActionReducer(FetchRequest);
+const request = buildApiActionReducer(FetchRequest);
 const saveRequest = buildApiActionReducer(SaveRequest);
```

Comment:
I don't think so, looking at the other references. I believe this was a merge error.